### PR TITLE
feat(InformationBlock): fix component style

### PR DESCRIPTION
**Pull request by BrunoGuimaraes103**
adjustment to the secondaryValue property, to include strike

Figma: https://www.figma.com/file/35iS2dwoogulJZPgFGKmCX/Advanced--Data-Card?type=design&node-id=521-104207&mode=design&t=vOgyqyOtcbGeKJwW-0

### CODE REVIEW

**Review by atabel — APPROVED** [review]

File: src/community/blocks.tsx [hunk 1]
```diff
@@ -118,7 +118,7 @@ export const InformationBlock: React.FC<InformationBlockProps> = ({
         <Inline space="between" alignItems="flex-end" aria-label={ariaLabel}>
             <BlockContent title={title} description={description} />
             <div className={classNames(styles.column, styles.rightContent)}>
-                <Text2 regular color={vars.colors.textSecondary}>
+                <Text2 regular color={vars.colors.textSecondary} decoration="line-through">
```
Comment (yceballost) [inline src/community/blocks.tsx:121]: Can you add Figma specs in the PR Body? I think this decoration has to be configurable

Comment (vinrosa) [reply src/community/blocks.tsx:121]: @yceballost This field is supposed to always use the line-through decoration, it's made to show the original price of the item.

**Review by yceballost — APPROVED** [review]
talked with @vinrosa. If in the future we need the configuration, we will add it later.

**Pull request merged**